ui/components-react/src/test/filter-builder/FilterBuilderState.test.ts

```diff
@@ -33,52 +35,43 @@ describe("usePropertyFilterBuilderState", () => {
   });
 
   it("initializes group with one empty rule", () => {
-    const { result } = renderHook(() => usePropertyFilterBuilderState());
-    const { state } = result.current;
-    expect(state.rootGroup).to.containSubset({
+    const { result } = renderHook(() => usePropertyFilterBuilder({}));
+    const { rootGroup } = result.current;
+    expect(rootGroup).to.containSubset({
       operator: PropertyFilterRuleGroupOperator.And,
       items: [
         {
-          groupId: state.rootGroup.id,
+          groupId: rootGroup.id,
         },
       ],
     });
   });
 
   it("adds rule to root group", () => {
-    const { result } = renderHook(() => usePropertyFilterBuilderState());
+    const { result } = renderHook(() => usePropertyFilterBuilder({}));
     const { actions } = result.current;
     actions.addItem([], "RULE");
 
-    const rootGroup = result.current.state.rootGroup;
-    expect(rootGroup).to.containSubset({
-      operator: PropertyFilterRuleGroupOperator.And,
-      items: [
-        {
-          groupId: rootGroup.id,
-        },
-        {
-          groupId: rootGroup.id,
-        },
-      ],
-    });
+    const { rootGroup } = result.current;
+    expect(rootGroup.items).to.have.lengthOf(2);
```

Comment:
Why `containSubset` was change to separate `expect` calls?

Reply:
Because `containSubset` is not working as expected with the same array values.

For example, this test

```ts
it.("my test", () => {
  expect([1]).to.containSubset([1, 1]);
});
```

would pass.

In this case, if something goes wrong and rootGroup would have only a single item with 
```ts
 groupId = rootGroup.id
```
the test would pass, while it should check if the root group has two items with the same group id.

Reply:
fixed